docs/freqai-configuration.md

```diff
@@ -43,10 +43,10 @@ The FreqAI strategy requires including the following lines of code in the standa
 
     def populate_indicators(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
 
-        # the model will return all labels created by user in `set_freqai_labels()`
```

Comment:
I'm sorry, I forgot that freqai is in a different repository than freqtrade. That is why I can't find set_freqai_labels(). I thought it was a deprecated function in freqtrade.

Reply:
Yeah, I think this PR no longer serve its intial purpose. Feel free to close it

Reply:
I'm just curious. Why are these differences between the docs and the actual code example? Does both correct?

Reply:
Could you please explain which are "these differences?"

Reply:
From [FreqaiExampleStrategy.py](https://github.com/freqtrade/freqtrade/blob/775ea1c8c657f8c6b41ad6604cdfd75b101e4f2c/freqtrade/templates/FreqaiExampleStrategy.py#LL232C8-L236C1): the model will return all labels created by user in **_feature_engineering__***
From [freqai-configuration.md](https://github.com/freqtrade/freqtrade/blob/775ea1c8c657f8c6b41ad6604cdfd75b101e4f2c/docs/freqai-configuration.md?plain=1#L46-L49): the model will return all labels created by user in _***set_freqai_labels()***_

From [FreqaiExampleStrategy.py](https://github.com/freqtrade/freqtrade/blob/775ea1c8c657f8c6b41ad6604cdfd75b101e4f2c/freqtrade/templates/FreqaiExampleStrategy.py#LL232C8-L236C1): the target mean/std values for each of the labels created by user in **_set_freqai_targets()_** for each training period.
From [freqai-configuration.md](https://github.com/freqtrade/freqtrade/blob/775ea1c8c657f8c6b41ad6604cdfd75b101e4f2c/docs/freqai-configuration.md?plain=1#L46-L49): the target mean/std values for each of the labels created by user in **_feature_engineering__*** for each training period.

Reply:
> Seems this should not change. Labels are created in `set_freqai_labels()`. Unless I miss something?
> 
> Could take this opportunity to simply improve the description entirely. In truth `self.freqai.start()` is returning predictions as columns in the `dataframe` where the column names match the keys created in `set_freqai_labels()`. And it is returning the target mean/std from the latest training period.
> 
> Something along these lines.

I'm still can't find `set_freqai_labels()`. Where is this function definition?

Reply:
[https://github.com/freqtrade/freqtrade/blob/775ea1c8c657f8c6b41ad6604cdfd75b101e4f2c/freqtrade/strategy/interface.py#LL702C18-L717C1
? ](https://github.com/freqtrade/freqtrade/blob/develop/freqtrade/strategy/interface.py#L702-L717)

?

Reply:
> https://github.com/freqtrade/freqtrade/blob/develop/freqtrade/strategy/interface.py#L702-L717 ? 

Wow set_freqai_targets() is the same as set_freqai_labels()! Thank you.

Reply:
the docs are using the wrong term (i guess in our mind, `set_freqai_targets` and `set_freqai_labels` is used interchangably.

the fix is however also wrong if you ask me - and should 1:1 replace `set_freqai_labels` with `set_freqai_targets` (you're expected to create the target / labels in `set_freqai_targets` - not in `feature_engineering_*` ).

Reply:
@xmatthias hence the https://github.com/freqtrade/freqtrade/blob/292bd62973011cb6c949984a5f6e8ab74d4cfd22/freqtrade/templates/FreqaiExampleStrategy.py#L233-L238 also wrong. Since I took it as the reference when making this PR